### ISSUE TITLE
None of the Tools are working in Ubuntu PPA build 1.0.6-0-201911301001~ubuntu18.04.1 when Experimental Input System is enabled

### ISSUE THREAD

**Issue by PaulMEdwards** [xournalpp/xournalpp#1645]
**Affects versions :**
 - OS: Ubuntu Linux `18.04.3` LTS
 - DE: Gnome `3.28.2` on X
 - libgtk version: `3.22.30-1ubuntu4`
 - Version of Xournal++: Package is labeled as `1.0.6-0~201911301001~ubuntu18.04.1` but the version listed in Help > About is `1.1.0+dev`

**Describe the bug**
Attempting to use any tool from the Toolbar or Tools menu results in no output on the page. This is true both when attempting to Annotate an existing PDF and also when attempting to manipulate a new Xournal document. Can't even use Hand tool to scroll the page(s) using click & drag.

Nothing obviously relevant appears in the console when running the application from there:
```
zii@zii-Latitude-5590:~$ xournalpp 
ALSA lib pcm.c:2495:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2495:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2495:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib pcm_route.c:867:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:867:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:867:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:867:(find_matching_chmap) Found no matching channel map
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
** Message: 10:39:19.073: lastOpenPath set
zii@zii-Latitude-5590:~$
```

When I remove the PPA version installed and install the Snap app available from the Ubuntu Software store (version `0+git.26c7205f` in Software store, `1.0.13` from Help > About), it works fine.

**To Reproduce**
1. Install build `1.0.6-0-201911301001~ubuntu18.04.1` from PPA
2. Run it and attempt to use any of the Tools on a document
3. Nothing happens (no output on document)
4. Uninstall PPA build and install Snap app from Ubuntu Software store and attempt to use it - works fine

**Expected behavior**
The Tools and all other basic functionality should work properly on the latest PPA build.

**Comment by LittleHuba** [comment]
Check out the settings -> Input System and make sure that the input classes are correct. 
Please provide input logs so we can fix the automatic assignment of the input classes.

**Comment by PaulMEdwards** [comment]
Thank you for the pointer, @LittleHuba.

I compared the Input System settings between versions listed above and found that the only difference is that the first option to enable the Experimental Input System was disabled on the version which works for me and enabled on the PPA build which does not.

I disabled that setting in the PPA build and restarted the application. The Tools now generally work on the PPA build... However, when I try to insert an Image, the application crashes with [these log messages](https://gist.github.com/PaulMEdwards/75e4a26381ac40a45ca36a19e651d854). Since that is a different matter, I would be glad to create a new separate issue for that problem.

Conversely, I enabled the Experimental Input System on the Ubuntu Store Snap version, restarted it, and the Tools continue to work fine. It doesn't crash when inserting an Image, either.

I have amended the title of this issue to reflect the information above.

**Comment by LittleHuba** [comment]
The old input system will be removed in the next release. Fixing this bug quickly  is therefore important.

Please provide the input logs. I won't be able to help otherwise.

**Comment by PaulMEdwards** [comment]
> Please provide the input logs. I won't be able to help otherwise.

Sorry, but I'm not sure where to find the input logs. I will gladly to provide them if you could direct me to where they are.

**Comment by PaulMEdwards** [comment]
Reviewing the logs and the settings revealed that the issue was painfully (and embarrassingly) obvious... The mouse input device was set to disabled and hence ignored within the editing area. Thanks for the help!

X++ is still crashing when I try to insert an image though, so I'll write that up as a new issue as I said I would. Just wanted to resolve this first to ensure it wasn't directly related.

**Comment by PaulMEdwards** [comment]
Image insert issue already documented in #1648 and resolved in #1650.
Confirmed change fixes issue for me as well.
